static/js/components/ranking_unit.tsx

```diff
@@ -48,6 +49,8 @@ interface RankingUnitPropType {
    * If true, only the ranks and labels will be shown and not the values.
    */
   hideValue?: boolean;
+  // For multi-column, these are the display strings for the value columns, in order.
+  svNames?: string[];
```

Comment:
instead of having 3 separate arrays for unit, scaling, and svNames where the values at each index has to match up, might be cleaner to have a single `columns?: { unit: string, scaling: number, name: string }[]`.

To avoid relying on array index, could even include an "svkey" for each column object & have RankingPoint.values be a map of svKey to value. 

These might be bigger changes so maybe add a todo if you agree that this is worthwhile

Reply:
i'll add a TODO